File: 02 - Even Fibonacci numbers/Go/main.go
package main

import "fmt"

func main() {

	var current int = 1
	var result int = 2
	var end = 4000000
	var temp int
	var array []int

	for result < end {
		temp = current + result
		current = result
		result = temp

		if (current % 2) == 0 {
			array = append(array, current)
		}
	}

	result = 0
	for _, v := range array {
		result += v
	}

	fmt.Println(result)
	fmt.Println(array)

}

File: 02 - Even Fibonacci numbers/Python/main.py
maxValue = 4000000

currentValue = 1
previousValue = 2

array = []

number = 0
result = 0

while previousValue < maxValue:
    print('Previous Value:', previousValue)
    temp = currentValue + previousValue
    currentValue = previousValue
    previousValue = temp

    if currentValue % 2 == 0:
        array.append(currentValue)

for number in array:
    result += number

print(result)